### ISSUE TITLE
tap2ntsvc error with py2exe 0.5.0

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7189]
|<img alt="njp's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| njp reported|
|-|-|
|Trac ID|trac#550|
|Type|defect|
|Created|2004-03-13 03:58:42Z|

<details><summary>Searchable metadata</summary>

```
trac-id__550 550
type__defect defect
reporter__njp njp
priority__high high
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__ 
keywords__ 
time__1079150322000000 1079150322000000
changetime__1474842546262285 1474842546262285
version__ 
owner__moonfallen moonfallen
cc__moonfallen cc__njp
```
</details>

**Comment by twisted-trac** [comment]
|<img alt="njp's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|njp commented|
|-|-|

```
#!html
<pre>
Attempting to use moonfallen sandbox tap2ntsvc with py2exe 0.5.0 generates
the error:

error: error in setup.cfg: command 'py2exe' has no such option 'service'

Python 2.3.3, Windows 2000, Twisted 1.2.0, py2exe 0.5.0

Notes: (1) samples provided with 0.5.0 (but not tested by me) use 'service',
           but also specify 'module' and not in a .cfg file (in setup.py
           directly);
        (2) general googling shows others with "no such option" issues with
            previously working py2exe options
</pre>
```

**Comment by twisted-trac** [comment]
|<img alt="moonfallen's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|moonfallen commented|
|-|-|

```
#!html
<pre>
Issue635 patched this; should be fixed now for py2exe 0.5.0.

Thank jml for the patch :)
</pre>
```

**Comment by twisted-trac** [comment]
|[<img alt="glyph's avatar" src="https://avatars.githubusercontent.com/u/716529?s=50" width="50" height="50">](https://github.com/glyph)<a name="note_3"></a>|@glyph commented|
|-|-|

In changeset 620b84494a0d59309d27c1228bc3886771051a32
```
#!CommitTicketReference repository="" revision="620b84494a0d59309d27c1228bc3886771051a32"
Merge pull request #550 from twisted/8839-refspec

Author: glyph

Reviewer: markrwilliams

Fixes: #8839

Fix git command in txchecker tox environment to avoid breaking certain checkouts.
```
